core/domain/exp_domain_test.py

```diff
@@ -344,6 +344,40 @@ def test_validation(self):
                 utils.ValidationError, 'destination ABC is not a valid'):
             exploration.validate()
 
+        # Ensure an answer group with two fuzzy rules is invalid
+        init_state = exploration.states[exploration.init_state_name]
+        default_outcome = init_state.interaction.default_outcome
+        default_outcome.dest = exploration.init_state_name
```

Comment:
Ah, I see what you were getting at now. I did ping you on wrapping up this PR in general, so feel free to get back if there might have still been communication issues.